graphs/util/db.py

```diff
@@ -1608,6 +1608,298 @@ def find_all_graphs_containing_nodes(uid, search_type, search_word, view_type, d
 
 	return graph_dict.values()
 
+
+def uploadGPMLFile(username, graphJSON, title):
+	'''
+		Uploads GPML file to GraphSpace via /upload.
+		@param username: Owner of graph
+		@param graphJSON: JSON of graph
+		@param title: Title of graph
+	'''
+
+	try:
+		# Create JSON stucture for GraphSpace recognized JSON
```

Comment:
Do you have anything specific in mind? I think the JSON spec is a good way to store the graph.

We can discuss this further on Monday :)